src/components/intern-app.html

```diff
@@ -3,135 +3,130 @@
-      <button class="add-button" type="button" on-click="createNewUserProfile">[[changeButtonText(addUserSelected)]]</button>
-    </template>
-    <user-list user-profiles="[[userProfiles]]" add-user-selected="[[addUserSelected]]"></user-list>
-  </template>
-  <script type="module">
-    'use strict'
-    class InternApp extends Polymer.Element {
-      static get is() { return 'intern-app'; }
+     .add-button {
+       background-color: #fca902;
+       box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+       padding: 20px;
+       margin-top: 0px;
+       font-size: 18px;
+       padding-right: 40px;
+       padding-left: 40px;
+       text-align: center;
+     }
+   </style>
+   <header>
+     <h1>Intern App</h1>
+   </header>
+   <template is="dom-if" if="[[!addUserSelected]]">
+     <button class="add-button" type="button" on-click="createNewUserProfile">[[changeButtonText(addUserSelected)]]</button>
+   </template>
+   <user-list user-profiles="[[userProfiles]]" add-user-selected="[[addUserSelected]]"></user-list>
+ </template>
+ <script type="module">
+   'use strict';
+   import localStorageApi from "../api/local-storage.js";
+   class InternApp extends Polymer.Element {
+     static get is() { return 'intern-app'; }
 
-      static get properties() {
-        return {
-          userProfiles: {
-            type: Array,
-            value: () => {
-              return [
-                { id: "001", firstName: 'Nick', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "002", firstName: 'Jack', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "003", firstName: 'Betty', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "004", firstName: 'Jordan', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "005", firstName: 'Brian', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "006", firstName: 'Gabby', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "007", firstName: 'John', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "008", firstName: 'Tony', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "009", firstName: 'Ryan', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" },
-                { id: "010", firstName: 'Mack', lastName: 'Cage', phoneNumber: 6413223232, email: "nick@nickcage.com" }
-              ];
-            }
-          },
-          addUserSelected: {
-            type: Boolean,
-            value: false
-          }
-        }
-      }
+     static get properties() {
+       return {
+         userProfiles: {
+           type: Array,
+           value: () => {
```

Comment:
can be simplified syntactically
```suggestion
           value: () => localStorageApi.getUsers()
```